grails-app/controllers/au/org/ala/ecodata/CommentController.groovy

```diff
@@ -0,0 +1,161 @@
+package au.org.ala.ecodata
+
+import grails.converters.JSON
+import groovy.json.JsonSlurper
+
+class CommentController {
+    def outputService
+
+    def ignores = ["action","controller"]
+
+    def list() {
+        List comments = []
```

Comment:
Interesting. I thought the implicit type conversion would have kicked in. What happens if you do ```int offset = params.start as Int```? If that doesn't work either then just ignore my comment.